bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/ReadEntryProcessor.java

```diff
@@ -71,6 +73,11 @@ protected void processPacket() {
         int errorCode = BookieProtocol.EOK;
         long startTimeNanos = MathUtils.nowInNano();
         ByteBuf data = null;
+        if (readTimeoutNanos >= 0 && startTimeNanos - enqueueNanos > readTimeoutNanos) {
```

Comment:
```suggestion
        if (readTimeoutNanos > 0 && startTimeNanos - enqueueNanos > readTimeoutNanos) {
```
I thinks <=0 for disabled is better:)